providers/remote-service.ts

```diff
@@ -0,0 +1,24 @@
+import { Constants } from './../../src/app/app.constants';
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+
+/*
+  Generated class for the QrCodeProvider provider.
+
+  See https://angular.io/guide/dependency-injection for more info on providers
+  and Angular DI.
+*/
+@Injectable()
+export class SharedRemoteService {
+    // TODO Issue #296: Use environments to manage different versions of APIs
+    getApiUrl: string = Constants.API_URL;
```

Comment:
We already have this constant in the apps using Shared Module? I believe so, but just to check

Reply:
What do you mean ? I'm referencing `Constants` application file indeed ... What I'm not sure of is if it is a good practice or not to reference application files from the shared module ...

Reply:
Yes, I feel strange depending on the app to have the correct value in the Constants file. It's passive of bugs. For now, should be okay but we need to think about it